<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Фон окна регистрации */
QDialog {
    background-color: #f0f0f0; /* Светло-серый фон */
}

/* Заголовок &quot;Регистрация&quot; */
QLabel#titleLabel {
    font-size: 40px; /* Большой шрифт */
    font-weight: bold; /* Жирный текст */
    color: #3498db; /* Синий цвет */
    padding: 20px; /* Отступы */
}

/* Подписи для полей ввода */
QLabel {
    font-size: 16px; /* Увеличенный шрифт */
    color: #2c3e50; /* Темно-синий цвет */
}

/* Поля ввода */
QLineEdit {
    padding: 8px; /* Внутренние отступы */
    border: 2px solid #bdc3c7; /* Серая рамка */
    border-radius: 8px; /* Закругленные углы */
    font-size: 14px; /* Размер шрифта */
    background-color: #ffffff; /* Белый фон */
    color: #2c3e50; /* Темно-синий текст */
}

QLineEdit:focus {
    border: 2px solid #3498db; /* Синяя рамка при фокусе */
}

/* Кнопка &quot;Зарегистрироваться&quot; */
QPushButton {
    background-color: #3498db; /* Синий цвет */
    color: white; /* Белый текст */
    border-radius: 8px; /* Закругленные углы */
    padding: 12px 24px; /* Внутренние отступы */
    font-size: 16px; /* Увеличенный шрифт */
}

QPushButton:hover {
    background-color: #2980b9; /* Темно-синий цвет при наведении */
}</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>451</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="email">
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="login">
      <property name="text">
       <string>Логин:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="confirmPassword">
      <property name="text">
       <string>Повторите пароль:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="emailLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="password">
      <property name="text">
       <string>Пароль:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="loginLineEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="nameLineEdit"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="name">
      <property name="text">
       <string>Имя:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="confirmPasswordLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="passwordLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="registerButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>299</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Зарегистрироваться</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="showPasswordCheckBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
    font-size: 10px; /* Размер шрифта */
    color: #2c3e50; /* Темно-синий цвет */
}</string>
   </property>
   <property name="text">
    <string>Показать пароль</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>-20</y>
     <width>341</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Регистрация</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
